src/org/labkey/test/components/ui/navigation/NavBar.java

```diff
@@ -48,6 +48,13 @@ public HasSearchResults searchFor(String searchString)
         return null;
     }
 
+    public FindByIdsDialog findByIds()
+    {
+        elementCache().findAndSearchMenuButton.click();
+        elementCache().findSamplesOption.click();
```

Comment:
Menu items usually show up right away so this should be fine (and the test are ok with it on TC).

Adding this:
` waitFor(()->elementCache().findSamplesOption.isDisplayed(), "Find samples menu did not show up.", 500);`
might help make the test more resilient to any random rendering slowness.